pkg/webhook/v1/managedcluster_validating.go

```diff
@@ -173,6 +184,29 @@ func (r *ManagedClusterWebhook) allowUpdateAcceptField(clusterName string, userI
 	return nil
 }
 
+// validateClusterNamespace checks the cluster namespace, if the namespace is terminating, reject the accept request.
+func (r *ManagedClusterWebhook) validateAcceptByClusterNamespace(clusterName string) error {
+	clusterNamespace, err := r.kubeClient.CoreV1().Namespaces().Get(context.TODO(), clusterName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
```go
if apierrors.IsNotFound(err) {
	return nil
}
if err != nil {
  return ...
}
```

Reply:
Fixed.

Reply:
Please take another look when you have time! Thanks!